app/code/Magento/Directory/etc/zip_codes.xml

```diff
@@ -196,7 +196,7 @@
     </zip>
     <zip countryCode="IL">
         <codes>
-            <code id="pattern_1" active="true" example="12345">^[0-9]{5}$</code>
+            <code id="pattern_1" active="true" example="6687865">^[0-9]{7}$</code>
```

Comment:
Could you replace example with realistic one please?

According to https://en.wikipedia.org/wiki/Postal_codes_in_Israel 5 digit zip codes are still widely used but unofficially, should we support both patterns for some time maybe?

Reply:
The example I used is a realistic one.
I think we should not support the old 5 digit format because it might cause a shippimg bounce.